systemtest/src/main/java/io/strimzi/systemtest/utils/specific/OlmUtils.java

```diff
@@ -100,4 +113,54 @@ public static String approveNonApprovedInstallPlanAndReturnDeploymentName(String
 
         return kubeClient().getCsv(namespaceName, csvName).getSpec().getInstall().getSpec().getDeployments().get(0).getName();
     }
+
+    public static Subscription prepareClusterOperatorSubscription(ClusterOperatorConfiguration clusterOperatorConfiguration) {
```

Comment:
Wouldn't having this builder/constructing method within the OlmInstallation? At least this seems to me like a template class as we have for many other classes (i.e., Kafka, KafkaBridge), you know. I am not sure we do anything like this in `Utils` or?

Reply:
Maybe I can create some template for that, but it's used even outside the OlmInstallation in the OLM upgrade. That's why I didn't want to put it there like that.